package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "raha-blockchain",
+  "version": "0.0.1",
+  "description": "Blockchain scripts for Raha.",
+  "main": "built/index.js",
+  "author": "Raha Foundation",
+  "license": "Apache-2.0",
+  "private": false,
+  "dependencies": {
+    "bs58": "^4.0.1",
+    "buffer": "^5.1.0",
+    "node-fetch": "^2.1.2",
+    "stellar-sdk": "^0.8.0"
+  },
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index.js"
+  },
+  "devDependencies": {
+    "@types/node-fetch": "^1.6.7",
+    "@types/stellar-sdk": "^0.8.1",
```

Comment:
Just be warned, it looks like the `stellar-sdk` typings are pretty threadbare, so it might not be totally accurate. That said perfectly good time to contribute to [`DefinitelyTyped`](https://github.com/DefinitelyTyped/DefinitelyTyped) if there are problems!